src/Uno.Sdk/targets/Uno.Common.Desktop.targets

```diff
@@ -8,8 +8,8 @@
 		<TargetPlatformIdentifier>Desktop</TargetPlatformIdentifier>
 		<TargetFrameworkIdentifier>Desktop</TargetFrameworkIdentifier>
 		<TargetPlatformSupported>true</TargetPlatformSupported>
-		<TargetPlatformVersion>$([System.Text.RegularExpressions.Regex]::Match($(TargetFramework), '\d+.\d+'))</TargetPlatformVersion>
-		<SupportedOSPlatformVersion Condition=" $(SupportedOSPlatformVersion) == '' ">8.0</SupportedOSPlatformVersion>
+		<TargetPlatformVersion>1.0</TargetPlatformVersion>
+		<SupportedOSPlatformVersion Condition=" $(SupportedOSPlatformVersion) == '' ">1.0</SupportedOSPlatformVersion>
```

Comment:
Are single quotes around `$(SupportedOSPlatformVersion)` not required?

Reply:
It's best indeed. Fixed.